sympy/physics/vector/frame.py

```diff
@@ -621,13 +621,16 @@ def _rot(axis, angle):
                 from sympy.physics.vector.functions import kinematic_equations
                 q1, q2, q3 = amounts
                 u1, u2, u3 = symbols('u1, u2, u3', cls=Dummy)
-                templist = kinematic_equations([u1, u2, u3], [q1, q2, q3],
-                                               rot_type, rot_order)
-                templist = [expand(i) for i in templist]
-                td = solve(templist, [u1, u2, u3])
-                u1 = expand(td[u1])
-                u2 = expand(td[u2])
-                u3 = expand(td[u3])
+                if all(i.is_number for i in [q1, q2, q3]):
```

Comment:
If it's a general symbol then it won't be clashing with any other symbol so that check should work. I guess the most definitive answer (since zeros can hide in t-dependent expressions) would be to use 
```
ui = list(symbols('u1, u2, u3', cls=Dummy))
for i, qi in enumerate((q1, q2, q3)):
    if qi.is_constant(t):
        ui[i] = 0
u1, u2, u3 = ui
```

(You'll have to define t as is done earlier in this section of code.)